### PR TITLE
updated gha instructions/workflows for quarto

### LINKED ISSUE #2743: r-lib/actions now install quarto by default
So can simplify quarto vignette advice and our workflows.

### PR DESCRIPTION
**Pull request by tanho63**
Closes #2743 

Updates quarto vignette and gha workflow files now that `setup-r-dependencies` automatically installs quarto as required.

### CODE REVIEW

**Comment by tanho63** [comment]
Some of the actions are failing for some reason, I think one that needs resolved in r-lib/actions?

**Comment by hadley** [comment]
@gaborcsardi any idea why we're missing quarto here?

**Comment by jayhesselberth** [comment]
You might try setting `install-quarto: true` in the `setup-r-dependencies` rule?

**Comment by tanho63** [comment]
It looks like it is trying to install quarto. Maybe it needs a parameter to also install tinytex? 
image

**Comment by jayhesselberth** [comment]
given tinytex was being installed by the quarto action that this PR removes, we may need to add:

```yaml
- uses: r-lib/actions/setup-tinytex@v2
```

File: DESCRIPTION [hunk 1]
```diff
@@ -1,6 +1,6 @@
 Package: pkgdown
 Title: Make Static HTML Documentation for a Package
-Version: 2.1.0.9000
+Version: 2.1.0.9001
```
Comment (jayhesselberth) [inline DESCRIPTION:3]: we don't typically bump the dev version between releases.

**Review by jayhesselberth — APPROVED** [review]
looks good to me. pkgdown needs to install tinytex to build one of the test vignettes, but most other packages shouldn't need it.

File: DESCRIPTION (hunk 1 above)
Comment (tanho63) [reply DESCRIPTION:3]:
```suggestion
Version: 2.1.0.9000
```

**Comment by hadley** [comment]
Perfect, thank you!

**Pull request merged**